packages/oauth/oauth-provider/src/account/account-manager.ts

```diff
@@ -144,30 +137,15 @@ export class AccountManager {
       throw InvalidRequestError.from(err, 'Account creation failed')
     })
 
-    // Only "remember" the newly created account if it was not created during an
-    // OAuth flow.
-    const remembered = requestId == null
-
-    const ephemeralCookie = remembered ? null : await randomHexId(32)
-
-    await this.upsertDeviceAccount(
-      deviceId,
-      account,
-      remembered,
-      requestId,
-      ephemeralCookie,
-    )
```

Comment:
I see we no longer upsert a device-account in account creation but we do still remove a device-account in the case of a failure.  Just double-checking these are both the intended behavior.